meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-firstBoot.sh

```diff
@@ -6,6 +6,8 @@ then
     mount /dev/mmcblk0p2 /tmp/once
     mount -o remount,rw /dev/mmcblk0p2 /tmp/once
     cp /etc/machine-id /tmp/once/etc
+    export MUID=$(dd if=/dev/random bs=1k count=1 2>/dev/null | sha256sum | awk '{print $1}')
+    echo "GP_PROBE_UUID=$MUID" > /tmp/once/etc/muid.data
```

Comment:
Approach seems ok but the values must match the expected UUID format: https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)